src/main/java/net/rptools/maptool/client/swing/SwingUtil.java

```diff
@@ -129,12 +117,48 @@ public static boolean isShiftDown(int modifiers) {
    * @param outerWindow window to be centered over
    */
   public static void centerOver(Window innerWindow, Window outerWindow) {
+    // how big can it be?
+    GraphicsConfiguration gc = outerWindow.getGraphicsConfiguration();
+    Insets insets = outerWindow.getToolkit().getScreenInsets(gc);
+    Rectangle bounds = gc.getDevice().getDefaultConfiguration().getBounds();
+    System.out.println(bounds);
+    System.out.println(insets);
+    bounds =
+        new Rectangle(
+            insets.left,
+            insets.top,
```

Comment:
It's not used, but for future sanity this should be `bounds.x + insets.left` and `bounds.y + insets.top`.